### DISCUSSION TITLE
Why are there users I don't know in my commit history on GitHub but not BitBucket?

### DISCUSSION THREAD

**Discussion by ackmanx** [community/community#22272] [New to GitHub]
<p>I just migrated my repositories from BitBucket. They are all committed from various personal computers I’ve had over the years. As such, I often did not always have the same git credentials in the commits being I’m the only one to ever commit on these.</p>
<p>So, it struck me as odd that there’s a random GitHub user on one of my projects listed as a contributor. Only GitHub thinks this, the same commit on BitBucket shows my name in the commit history.</p>
<p>What’s going on?</p>
<p>GitHub: <a href="https://github.com/ackmanx/photo-scripts/commits/master" rel="nofollow noopener">https://github.com/ackmanx/photo-scripts/commits/master</a></p>
<p>BitBucket: <a href="https://bitbucket.org/ackmanx/photo-scripts/commits/all" rel="nofollow noopener">https://bitbucket.org/ackmanx/photo-scripts/commits/all</a></p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Most likely this is because one of your commits was under an email address that is associated with a different GitHub account. See the Support Protips article <a href="https://github.community/t/why-is-my-commit-associated-with-the-wrong-person/10199">Why is my commit associated with the wrong person?</a> for more information.</p>
<p>Let us know if you have more questions.</p>
